src/tcgwars/logic/impl/gen8/RebelClash.groovy

```diff
@@ -131,7 +131,7 @@ public enum RebelClash implements LogicCardInfo {
   BINACLE_103 ("Binacle", "103", Rarity.HOLORARE, [POKEMON, BASIC, _FIGHTING_]),
   BARBARACLE_104 ("Barbaracle", "104", Rarity.HOLORARE, [POKEMON, EVOLUTION, STAGE1, _FIGHTING_]),
   ROLYCOLY_105 ("Rolycoly", "105", Rarity.HOLORARE, [POKEMON, BASIC, _FIGHTING_]),
-  CARKOAL_106 ("Carkoal", "106", Rarity.HOLORARE, [POKEMON, EVOLUTION, STAGE1, _FIGHTING_]),
```

Comment:
You can't change the Enum (CARKOAL_106) it will cause errors because previously created decks will no longer be able to find the card.